README.md

```diff
@@ -2,23 +2,15 @@
 
 ![alt tag](https://raw.githubusercontent.com/tripal/tripal/7.x-3.x/tripal/theme/images/tripal_logo.png)
 
-Tripal is a toolkit for construction of online biological (genetics, genomics,
-breeding, etc), community database, and is a member of the 
-[GMOD](http://www.gmod.org) family of tools. Tripal v3 provides by default
-integration with the [GMOD Chado database](http://gmod.org/wiki/Chado_-_Getting_Started).
-Tripal's primary goals are: 
+Tripal is a toolkit for constructing online biological (genetics, genomics, breeding, etc.) community database, and Tripal is a member of the [GMOD](http://www.gmod.org) family of tools. **Tripal v3** provides integration with the [GMOD Chado database](http://gmod.org/wiki/Chado_-_Getting_Started) by default.
 
-Genomics, genetics, breeding and other biological data are increasingly complicated and time consuming to publish online for other researchers to search, browse and make discoveries.   Tripal provides a framework to reduce the complexity of creating such a site, and provides access to a community of similar groups that share community-standards, and interact to address questions and learn best practices for sharing, storing and visualizing complex biological data.
+Genetics, genomics, breeding, and other biological data are complicated and time-consuming to publish online for others to search, browse and discover. Tripal provides not only a framework to reduce the complexity, but also access to groups that share community-standards.  They are encouraged to interact to address questions and learn the best practices for sharing, storing, and visualizing biological data.
```

Comment:
* leave in "increasingly".   The idea is that biological data becomes more and more complex as technologies evolve.

* "discover" isn't quite right.  " make discoveries." should probably have been "make discoveries with".

Reply:
Hi @bradfordcondon , 
Thank you again for reviewing the changes.  I've updated most of the changes but would you please clarify the following before I commit my changes?
* "They are encouraged to interact to address questions and learn the best practices for sharing, storing, and visualizing biological data."
  * "They" - should I change it to "The users of Tripal"?
  * I removed "complex" because it is clear that biological data is complex as it has been mentioned earlier in the paragraph. Though, I can put it back so the complexity of biological data is emphasized.